### ISSUE TITLE
Vertical filtering doesn't work

### ISSUE THREAD

**Issue by spotUP** [mrchrisster/MiSTer_SAM#234]
Hello i have configured SAM to only show vertical games, but it seems to ignore it?

I have this in my ini file:

# --------- ARCADE ORIENTATION ---------
# Set Arcade orientation to "horizontal" "vertical" or leave empty
arcadeorient="vertical"

**Comment by mrchrisster** [comment]
I have actually never tried this option myself but it works the same way as horizontal filtering. Have you tried restarting your MiSTer? You can also try to reset gamelists in the menu.

**Comment by mrchrisster** [comment]
Any updates on this?

**Comment by spotUP** [comment]
Yes I did, didn’t work. I ended up deleting all horizontal games in the end. Not optimal. :)//JSkickat från min iPhone16 aug. 2024 kl. 20:35 skrev Christoph Helms ***@***.***>:﻿
I have actually never tried this option myself but it works the same way as horizontal filtering. Have you tried restarting your MiSTer? You can also try to reset gamelists in the menu.

**Comment by mrchrisster** [comment]
Thanks for checking. I found an error and believe this should be working correctly now. Care to give it a try?
